controllers/managedfusionoffering_controller.go

```diff
@@ -46,18 +50,33 @@ type ManagedFusionOfferingReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.6.4/pkg/reconcile
 func (r *ManagedFusionOfferingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = context.Background()
-	_ = r.Log.WithValues("managedfusionoffering", req.NamespacedName)
+	log := r.Log.WithValues("managedfusionoffering", req.NamespacedName)
+	log.Info("Starting reconcile for ManangedFusionOffering")
 
-	// TODO(user): your logic here
+	managedFusionOfferingList := &v1alpha1.ManagedFusionOfferingList{}
+	if err := r.Client.List(ctx, managedFusionOfferingList); err != nil {
+		return ctrl.Result{}, err
+	}
+	if len(managedFusionOfferingList.Items) == 0 {
```

Comment:
am I correct in assuming that for each offering we'll have one offering CR?

Reply:
Yes, correct

Reply:
Just for clarity, assume we are supporting offering X and Y, both offerings can co-exist on the same cluster.
Then there will be two MFO CRs on the cluster, one for offering X and another for Y.